Cannot build Config, some of required attributes are not set [uiDir] (Consul 0.8)

With Consul 0.8 the uiDir attribute isn't needed anymore (Consul actually gives alerts about this in its logging if you do) - but removing that parameter gives an error in this client when requesting the agent (AgentClient.getAgent)

The error that is thrown:
Caused by: java.lang.IllegalStateException: Cannot build Config, some of required attributes are not set [uiDir]
        at com.orbitz.consul.model.agent.ImmutableConfig$Builder.build(ImmutableConfig.java:2513)
        at com.orbitz.consul.model.agent.ImmutableConfig.fromJson(ImmutableConfig.java:2007)


Reason for this - in com.orbitz.consul.model.agent.Config the uiDir-attribute is configured as a required property.

--- a/src/main/java/com/orbitz/consul/model/agent/Config.java
+++ b/src/main/java/com/orbitz/consul/model/agent/Config.java
@@ -92,7 +92,7 @@ public abstract class Config {
     public abstract List<String> getStartJoin();
 
     @JsonProperty("UiDir")
-    public abstract String getUiDir();
+    public abstract Optional<String> getUiDir();
 
     @JsonProperty("PidFile")
     public abstract String getPidFile();

